### ISSUE TITLE
Georeference Extension

### ISSUE THREAD

**Issue by glenrobson** [IIIF/trc#109]
## Links

 * Original Issue: https://github.com/IIIF/iiif-stories/issues/129
 * Pull Request: https://github.com/IIIF/api/pull/2082
 * Preview: https://preview.iiif.io/api/georef/extension/georef/

## Background and Summary

This extension is to formalise the annotation that allows the Georeferencing of a IIIF image.  Georeferencing requires the following information:

 *  mask of image (area to be georef, can be whole image)
 *  control points related to geographic space + pixel coordinate space
 *  and a transformation algorithm

The mask is achieved by using a SVG image selector on the IIIF image. The control points and transformation algorithm are stored in the body of the annotation in GeoJson. The control points use a custom property called resourceCoords and a standard GeoJson property geometry to tie the two different coordinate systems together. 

There is a tool called allmaps which has been used to prototype the development of the spec:

https://editor.allmaps.org/

## Proposed Solution

The Maps TSG group would like this extension to be approved.

**Comment by azaroth42** [comment]
There's still an awful lot of `[@someone do something]` in the preview link. Is this the most recent?

**Comment by thehabes** [comment]
Yes, it is, we were a little behind the optimal timeline.  We will knock them out and be done with changes in the next few hours.  Apologies, we know that is a bit disruptive.  I will add another note here so you can be assured we aren't making changes anymore while you all try to review it.

**Comment by thehabes** [comment]
Thanks for bearing with us everyone!  We synced up and all of our intended changes are in.  We will not make anymore changes until after this round of the TRC process.

**Comment by triplingual** [comment]
Tiny: The appendix section should have its heading respelled to "Appendices"

**Comment by triplingual** [comment]
It's a little far along in the process, I know, but it would be good to start having more examples in specs from outside Europe + North America, and from institutions outside Europe + North America. Is there any way there could be a little work to swap the example in the full canvas for an example from a different continent? (Or could be done immediately after this issue is completed?)

**Comment by bertspaan** [comment]
@triplingual yes, very good idea! Let me know if you know of IIIF-accessible map collections from institutions outside of Europe + North America. Or add them to https://github.com/allmaps/iiif-map-collections/blob/master/iiif-map-collections.yml with a PR.

**Comment by zimeon** [comment]
Intro, para 2, missing "CG" but perhaps better still to spell out Community Group and Technical Specification Group:
> Through the work of the [IIIF Maps **Community Group**](https://iiif.io/community/groups/maps/) and [IIIF Maps Technical **Specification Group**](https://iiif.io/community/groups/maps-tsg/) ~groups~, a commonality of techniques to georeference IIIF Canvases and Images in the context of a global map became evident, and a desire to have standards and best practices for georeferencing became known.

**Comment by regisrob** [comment]
A few remaining typos:
`geoerferenced`
`This is extension is aimed at`
`examples with robuse implementation guidance`
`6. Appendecies`
`perserverance`

**Comment by thehabes** [comment]
Thank you everyone, your time is appreciated.  The extension authors will meet early next week to go over the comments and suggestions.  We will make changes and respond to individuals in cases where that seems appropriate.

**Comment by glenrobson** [comment]
### Issue 109 (Georeference Extension)
  +1: 25 [akrishnan15 cubap digitaldogsbody dlpierce eliotjordan emilyewahl glenrobson hadro irv jcreel jpadfield jtweed julsraemy ksclarke markpatton markpbaggett mikeapp mixterj nfreire regisrob robcast shuddles thehabes triplingual zimeon]
   0: 0 []
  -1: 0 []
  Not TRC: 0 []
  Ineligible: 2 [azaroth42 ioanrichards]

### Result: 25 / 25 = 1.00
Super majority is in favor, issue is approved
